verl/trainer/ppo/core_algos.py

```diff
@@ -467,6 +467,10 @@ def kl_penalty(logprob: torch.FloatTensor, ref_logprob: torch.FloatTensor, kl_pe
     if kl_penalty == "mse":
         return 0.5 * (logprob - ref_logprob).square()
 
+    if kl_penalty == "k3_off_policy":
```

Comment:
Could you give a reference here?

Reply:
Sure, no problem. I wonder if Verl can provide some computing support to test the performance within Verl.